music21/roman.py

```diff
@@ -832,6 +859,11 @@ def romanNumeralFromChord(chordObj,
         'i64b3': 'Sw43',
         'io6b5b3': 'Ger65',
     }
+    secondaryDominantSwaps = {
+        ('#', 'i'): ('vii', 'ii'),
+        ('#', 'iv'): ('vii', 'V'),
```

Comment:
same as above.  Has this been tested with diminished chords? viio7 of V, etc.  I imagine it would work, but need to check